includes/class-assets.php

```diff
@@ -180,6 +180,12 @@ public function enqueue_scripts() {
 				WPBDP_VERSION
 			);
 		}
+
+		// Enable `grunt-contrib-watch` livereload.
+		// Live reload server will be started with the watch task per target.
+		if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG && in_array( $_SERVER['REMOTE_ADDR'], [ '127.0.0.1', '::1' ] ) ) {
+			wp_enqueue_script( 'livereload', 'http://localhost:35729/livereload.js?snipver=1', [], WPBDP_VERSION, true );
```

Comment:
Can we use the site url for this? And where does the livereload.js come from? Is it created with an npm command?

Reply:
No! It generates by Grunt in [this line](https://github.com/Strategy11/business-directory-plugin/pull/289/files#diff-8e7e5f941c67b0cf481a9d650295119d4922b96117e7adc069ba30a81a78c65fR90) when we run the `grunt watch` command with this fixed URL.

Reply:
Oooh!! This is really cool! I didn't test it before, and I'm not sure why that url works, but it's magical.